app/webpacker/src/stylesheets/_feedback_useful_banner.scss

```diff
@@ -0,0 +1,45 @@
+.feedback-useful-banner {
+  border-top: 1px solid $govuk-border-colour;
+
+  .feedback-useful-container {
+    position: relative;
+    padding: 1em;
+    background-color: govuk-colour("light-grey");
+    height: 36px;
+    padding-top: 17px;
+  }
+
+  p {
+    font-weight: bold;
+    float: left;
+    padding-top: 7px;
+    padding-left: 11px;
+    padding-right: 15px;
+  }
+
+  .govuk-button {
+    border: 1px solid govuk-colour("black");
+    box-shadow: 0 2px 0 govuk-colour("black");
+    min-width: 6em;
+  }
+}
+
+.govuk-footer {
+  border-top: 10px solid govuk-colour("blue");
+}
+
+@media (max-width: 900px) {
```

Comment:
Is it possible to utilise the existing govuk break points rather than adding new ones? eg https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/webpacker/src/stylesheets/_tables.scss#L185

Reply:
I tried using Gov UK breakpoints but this is quite unique and the ones I tried didn't match my needs.

Reply:
just updated 640 to use existing CSS breakpoint, we have to stick 900px on line 31